walkthrough.md

```diff
@@ -179,6 +184,8 @@ Note the pipe `|`!
 
 - What is two-way data binding?
 
+<!-- AM: The more I think about it there should be a CFU for the view model. Reviewing it'll help. -->
+
```

Comment:
I dunno. I kind of want to leave it at "it's an instance of a controller", and I don't care too much if they get that either. I don't really even know what a viewmodel is, and don't want to muddy the waters. Do you have a definition in mind?